app/lib/scihist_digicoll/env.rb

```diff
@@ -288,6 +284,18 @@ def self.shrine_dzi_storage
     # Used (infrequently) by additional job servers whose only purpose is to handle special tasks:
     define_key :special_job_worker_count, default: 0
 
+
+    # OUTGOING EMAIL:
+
+    # From address:
+    define_key :no_reply_email_address
+    # To addresses (these are email lists maintained by IT.)
+    define_key :digital_tech_email_address
+    define_key :digital_email_address
+
+    # These are the credentials used for accessing Amazon's simple email server:
+    define_key :ses_smtp_username
+    define_key :ses_smtp_password
     define_key :smtp_host
```

Comment:
These values would apply even if it wasn't `ses`, right? Change variable names to just `smtp_username` and `smtp_password`, what do you think?

Reply:
Absolutely, duh!